src/main/java/no/nav/pto/veilarbportefolje/cv/CVService.java

```diff
@@ -0,0 +1,55 @@
+package no.nav.pto.veilarbportefolje.cv;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.arbeid.cv.avro.Melding;
+import no.nav.arbeid.cv.avro.Meldingstype;
+import no.nav.common.types.identer.AktorId;
+import no.nav.pto.veilarbportefolje.elastic.ElasticServiceV2;
+import no.nav.pto.veilarbportefolje.kafka.KafkaConfig;
+import no.nav.pto.veilarbportefolje.kafka.KafkaConsumerService;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+
+@RequiredArgsConstructor
+@Service
+@Slf4j
+public class CVService implements KafkaConsumerService<Melding> {
+    private final ElasticServiceV2 elasticServiceV2;
+    private final CvRepository cvRepository;
+    private final AtomicBoolean rewind = new AtomicBoolean(false);
+
+    private boolean cvEksistere(Melding melding) {
+        return melding.getMeldingstype() == Meldingstype.ENDRE || melding.getMeldingstype() == Meldingstype.OPPRETT;
+    }
+
+    @Override
+    public void behandleKafkaMelding(Melding kafkaMelding) {
+        log.info(
+                "Behandler kafka-melding med key {} på topic {}",
+                kafkaMelding.getAktoerId(),
+                KafkaConfig.Topic.CV_ENDRET
+        );
+        AktorId aktoerId = AktorId.of(kafkaMelding.getAktoerId());
+
+        if (cvEksistere(kafkaMelding)) {
+            cvRepository.upsertCvEksistere(aktoerId, true);
+            elasticServiceV2.updateCvEksistere(aktoerId, true);
+        } else {
+            cvRepository.upsertCvEksistere(aktoerId, false);
+            elasticServiceV2.updateCvEksistere(aktoerId, false);
```

Comment:
Can't this be simplified?
 cvRepository.upsertCvEksistere(aktoerId, cvEksistere(kafkaMelding))
 elasticServiceV2.updateCvEksistere(aktoerId, cvEksistere(kafkaMelding))

Reply:
Or:
boolean cvcvEksistere = cvEksistere(kafkaMelding)
cvRepository.upsertCvEksistere(aktoerId, cvEksistere)
elasticServiceV2.updateCvEksistere(aktoerId, cvEksistere)

Reply:
Nice, its more simple and more readable. Thanks!